build/deps/v8.bzl

```diff
@@ -28,7 +28,7 @@ PATCHES = [
     "0020-Remove-unneded-latomic-linker-flag.patch",
     "0021-Add-methods-to-get-heap-and-external-memory-sizes-di.patch",
     "0022-Remove-DCHECK-from-WriteOneByteV2-to-skip-v8-fatal.patch",
-    "0023-Add-more-sandbox-APIs.patch",
+    "0023-Modify-where-to-look-for-dragonbox.patch",
```

Comment:
Just confirming we no longer need the Add-more-sandbox-APIs patch

Reply:
Yes it's upstreamed.